package.json

```diff
@@ -39,50 +38,50 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "bunsen-core": "0.18.1",
-    "ember-browserify": "1.1.13",
-    "ember-bunsen-core": "0.12.1",
-    "ember-cli": "^2.7.0",
+    "bunsen-core": "~0.18.2",
+    "ember-browserify": "^1.1.13",
+    "ember-bunsen-core": "~0.12.2",
+    "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^2.0.0",
+    "ember-cli-chai": "~0.2.0",
     "ember-cli-code-coverage": "0.3.4",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.4",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-mirage": "0.2.1",
-    "ember-cli-mocha": "0.10.4",
+    "ember-cli-mocha": "~0.13.0",
     "ember-cli-moment-shim": "2.2.1",
     "ember-cli-notifications": "^4.0.4",
     "ember-cli-showdown": "^2.5.0",
     "ember-cli-test-loader": "1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-computed-decorators": "0.2.2",
-    "ember-concurrency": "0.7.15",
-    "ember-data": "^2.7.0",
+    "ember-computed-decorators": "^0.2.2",
+    "ember-concurrency": "^0.7.15",
+    "ember-data": "~2.8.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-disable-proxy-controllers": "^1.0.1",
-    "ember-elsewhere": "0.4.1",
+    "ember-elsewhere": "~0.4.1",
     "ember-export-application-global": "^1.1.1",
-    "ember-frost-core": "0.30.1",
+    "ember-frost-core": "^1.0.0",
     "ember-frost-demo-components": "1.0.0",
-    "ember-frost-fields": "1.0.0",
-    "ember-frost-popover": "0.1.0",
-    "ember-frost-tabs": "^3.2.0",
+    "ember-frost-fields": "^4.0.0",
+    "ember-frost-popover": "^4.0.0",
+    "ember-frost-tabs": "^4.0.0",
```

Comment:
Bunsen uses `ember-frost-tabs` to provide tabs from the root `cells` array in a bunsen view.